Keypad/Keypad.js

```diff
@@ -174,32 +174,65 @@ const KeypadBase = kind({
 		className: 'keypad'
 	},
 
-	render: ({disabled, onKeyButtonClick, spotlightDisabled, ...rest}) => {
+	computed: {
+		keys: ({isCallActive, skin}) => {
+			const callStatus = isCallActive ? 'calldecline' : 'callaccept';
+			const editStatus = isCallActive ? 'keypad' : 'backspace';
+
+			return [
+				{text: '1'},
+				{text: '2', label: 'abc'},
+				{text: '3', label: 'def'},
+				{text: '4', label: 'ghi'},
+				{text: '5', label: 'jkl'},
+				{text: '6', label: 'mno'},
+				{text: '7', label: 'pqrs'},
+				{text: '8', label: 'tuv'},
+				{text: '9', label: 'wxyz'},
+				{text: '*'},
+				{text: '0'},
+				{text: '#'},
+				{icon: skin !== 'silicon' ? 'phone' : callStatus},
+				{icon: skin !== 'silicon' ? 'arrowuturn' : editStatus}
+			];
+		}
+	},
+
+	render: ({disabled, keys, onKeyButtonClick, spotlightDisabled, ...rest}) => {
+		delete rest.isCallActive;
+
 		return (
 			<SpotlightContainerLayout {...rest} align="center end" className={css.keypad} inline spotlightDisabled={spotlightDisabled} wrap>
-				{KEY_LIST.map((keyText, rowIndex) => {
+				{keys.map((keyText, rowIndex) => {
 					const {icon, text} = keyText;
-					const isIcon = icon === 'arrowuturn' || icon === 'phone';
 
-					let ariaLabel = text;
+					let ariaLabel;
 					if (icon === 'arrowuturn') {
 						ariaLabel = $L('backspace');
 					} else if (icon === 'phone') {
 						ariaLabel = $L('call');
 					}
 
+					let color = '';
+					if (icon === 'callaccept') {
+						color = 'accept';
+					} else if (icon === 'calldecline') {
+						color = 'decline';
+					}
+
 					return (
 						<Cell
-							aria-label={ariaLabel}
+							aria-label={text ? text : ariaLabel}
+							className={css[`${color}`]}
```

Comment:
How about use the icon variable directly without `color`.
Need to change class name to callaccept and calldecline from accept and decline respectively;

```suggestion
							className={css[icon]}							
```